doc/en/developer/source/policies/service_providers.rst

```diff
@@ -0,0 +1,91 @@
+.. _service_providers:
+
+Service Providers
+=================
+
+Service providers are an important part of GeoServer community, thank you for helping GeoServer reach a larger audience. Just as open source is a part of your success, your participation and investment is fundamental to the GeoServer project's sustainability.
+
+The PSC manages:
+
+* `GeoServer Commercial Support <http://geoserver.org/support/>`__ (geoserver.org)
+* `Service Providers List <https://www.osgeo.org/service-providers/?p=geoserver>`__ (osgeo.org)
+  
+  Available from `project page <https://www.osgeo.org/projects/geoserver/>`__ from *view all service providers* link.
+
+In addition:
+
+* Project `sponsorship <https://github.com/geoserver/geoserver/wiki/Sponsor>`__ is another way to increase visibility, with specific activities available to sponsor throughout the year. Financial support is also recognized on the `osgeo sponsor <https://www.osgeo.org/sponsors/>`__ page.
+
+Service Provider checklist
+--------------------------
+
+To recognizing experienced providers and core contributors consistently we have developed the following checklist.
+
+Core contributors:
+
+* ``core`` demonstrating an ongoing responsibility: security email list, releases, supported extension
+* ``committer`` active development: geoserver-devel, review of pull-requests, assigned jira tickets, development activities such bug-stomps and sprint participation
+* ``support`` sustained community support: geoserver-user, review of jira, gis.stackexchange.com, something!
+
+Experienced providers: work on GeoServer when hired by customers but are not always active
```

Comment:
However experienced providers are doing only some of these, right?

Reply:
The goal here is to allow folks to demonstrate (preferability with a link) the kind of work they are good at.

Experienced providers may do some of these, and even help out with releases on occasion, the difference is they are not taking on ongoing responsibility.